### ISSUE TITLE
KM100 setup failed

### ISSUE THREAD

**Issue by Bambie07** [bosch-thermostat/home-assistant-bosch-custom-component#436]
**Describe the bug**
My integration suddenly stopped working a few weeks ago. I don't know if this was after an update of home-assistant.

**To Reproduce**
Use `docker run -it test scan -i --host=192.168.44.19 --token=4XXXXXX7 --password=eXXXXXXM --protocol HTTP --device IVT`. Get an error with `Successfully connected to gateway. Found UUID: None`. Searching in output json there are correct reading e.g. of outdoor temperatures. These are not shown in home assistant. With home-assistant I'm able to add the device, but it shows 'Unknown model' and all entities has no data.

**Expected behavior**
Correct reading of data.

**Version**
  - HA version 2024.9.0
  - HA Bosch component version? 0.26.1
  - Gateway info of ios app shows firmware version 04.08.02
  - Python Bosch HTTP client version? 

** Debug SCAN **
http://jsonblob.com/1282379029221859328

If you need additional information, please let me know. I've written some python code, if someone describes, where to add the new device, I can look into it myself

**Comment by maschue** [comment]
I was having a similar issue with km200, suddenly there were no heating circuits and no water heaters available anymore. I'm not aware about any updates (neither on HA nor on the gateway itself). 
The Buderus App was affected as well, first I was having trouble to log in at the gateway, and when I succeeded, no heaters were displayed. 
I did a power reset on the gateway and on the heater, and after some minutes all sensors re-appeared in the App. 
After a reboot of HA everything worked fine again.
